common/src/main/resources/assets/figura/lang/en_us.json

```
@@ -907,18 +908,18 @@
     "figura.docs.entity.is_sneaking": "Returns true if this entity is logically sneaking (can't fall from blocks edges, can't see nameplate behind walls, etc)",
     "figura.docs.entity.is_crouching": "Returns true if this entity is visually sneaking",
     "figura.docs.entity.get_item": "Gets an ItemStack for the item in the given slot\nFor the player, slots are indexed with 1 as the main hand, 2 as the off hand, and 3,4,5,6 as the 4 armor slots from the boots to the helmet\nIf an invalid slot number is given, this will return nil",
-    "figura.docs.entity.get_nbt": "Gets a table containing the NBT of this entity\nPlease note that not all values in the entity's NBT may be synced, as some are handled only on server side",
+    "figura.docs.entity.get_nbt": "Gets a table containing the NBT of this entity\nPlease note that not all values in the entity's NBT may be synced, as some are handled only on the server side",
     "figura.docs.entity.is_on_fire": "Returns true if this entity is currently on fire",
     "figura.docs.entity.is_alive": "Returns whether this entity is alive or not",
     "figura.docs.entity.get_permission_level": "Returns the permission level number of this entity\nServer Operators, by default, have the permission level of 4",
-    "figura.docs.entity.get_passengers": "Returns a List of entities of all passengers this entity have",
+    "figura.docs.entity.get_passengers": "Returns a List of entities of all passengers this entity has",
     "figura.docs.entity.get_controlling_passenger": "Returns the entity that is controlling this entity",
     "figura.docs.entity.get_controlled_vehicle": "Return the vehicle that this entity is controlling",
     "figura.docs.entity.has_container": "Checks if the entity has a container (Chest Boats, Minecarts with Chests, …)",
     "figura.docs.entity.has_inventory": "Checks if the entity has an inventory (Horses, Camels, LLamas, …)",
-    "figura.docs.entity.get_targeted_block": "Returns a proxy for your currently targeted BlockState\nThis BlockState appears on the F3 screen\nMaximum and Default distance is 20, Minimum is -20\nReturns a vararg of the block, the hit position and the block face the hit collided",
+    "figura.docs.entity.get_targeted_block": "Returns a proxy for your currently targeted BlockState\nThis BlockState appears on the F3 screen\nMaximum and Default distance is 20, Minimum is -20\nReturns as multiple results of the block, the hit position and the block face the hit collided",
```

A few changes to make this flow slightly better. The last line didn't flow very well.
Despite my changes I still don't feel confident about the last line, maybe you or someone else could have another look at it.
```suggestion
    "figura.docs.entity.get_targeted_block": "Returns a proxy for your currently targeted BlockState\nThis BlockState appears on the F3 screen\nThe maximum (and default) distance is 20, minimum is -20\nReturns the block, the hit position, and the targeted block face as three separate values",
```

---

Yeah I agree, this does flow better. I'd like another opinion tho.

---

this looks good to me